tests/test_utils/tests.py

```diff
@@ -812,24 +812,127 @@ class AssertFieldOutputTests(SimpleTestCase):
 
     def test_assert_field_output(self):
         error_invalid = ['Enter a valid email address.']
-        self.assertFieldOutput(EmailField, {'a@a.com': 'a@a.com'}, {'aaa': error_invalid})
+        self.assertFieldOutput(forms.EmailField, {'a@a.com': 'a@a.com'}, {'aaa': error_invalid})
         with self.assertRaises(AssertionError):
-            self.assertFieldOutput(EmailField, {'a@a.com': 'a@a.com'}, {'aaa': error_invalid + ['Another error']})
+            self.assertFieldOutput(forms.EmailField,
+                                   {'a@a.com': 'a@a.com'}, {'aaa': error_invalid + ['Another error']})
         with self.assertRaises(AssertionError):
-            self.assertFieldOutput(EmailField, {'a@a.com': 'Wrong output'}, {'aaa': error_invalid})
+            self.assertFieldOutput(forms.EmailField, {'a@a.com': 'Wrong output'}, {'aaa': error_invalid})
         with self.assertRaises(AssertionError):
             self.assertFieldOutput(
-                EmailField, {'a@a.com': 'a@a.com'}, {'aaa': ['Come on, gimme some well formatted data, dude.']}
+                forms.EmailField, {'a@a.com': 'a@a.com'}, {'aaa': ['Come on, gimme some well formatted data, dude.']}
             )
 
     def test_custom_required_message(self):
-        class MyCustomField(IntegerField):
+        class MyCustomField(forms.IntegerField):
             default_error_messages = {
                 'required': 'This is really required.',
             }
         self.assertFieldOutput(MyCustomField, {}, {}, empty_value=None)
 
 
+class AssertFormValidTests(SimpleTestCase):
+    class TestForm(forms.Form):
+        name = forms.CharField(label='Name', max_length=4)
+        age = forms.IntegerField(label='Age')
+
+    form_cls = TestForm
```

Comment:
No need to define another attribute. The form class should be accessible as `self.TestForm`.